### DISCUSSION TITLE
Nouvelle Arme et Event

### DISCUSSION THREAD

**Discussion by AlexisLhotte** [Crownicles/Crownicles#1297] [Suggestions fermées]
Prévoir un event "Sandwich Sacré" et une arme Sandwich Sacré pour le draftbot

**Comment by Nargacaura** [comment]
Je ne pense pas que cette idée soit acceptée pour 2 raisons:
a. L'event est suggéré dans le mauvais répo (il peut être suggéré dans le répo **Events**, et pas ici, mais faut passer par la discussion des rédacteurs et traducteurs, accessible via l'un des 2 rôles au moins sur Discord),
b. Pour proposer un objet, il est requis de **faire un don de 5€ ou plus au bot**. Un item, ça ne se propose pas comme ça.

**Reply by BastLast** [reply]
Le don n'est pas obligatoire pour la proposition d'item 😁👍 (juste faut que l'item soit vraiment cool pour que la suggestion soit traitée et là il ne l'est pas.
